### ISSUE TITLE
Feature Request: Mail Adress Save to Adressbook

### ISSUE THREAD

**Issue by WASDIT** [modoboa/modoboa-webmail#100]
If you enter in the webmail recipient field a mail adress that you have the possiblity to save them as Contact in the adressbook.

**Comment by blokkie** [comment]
:+1: 
A lot of times I've  missed this option.  And I'm not getting used to it.  All to many times I have to go back to the original mail and save the contact there , then go back to the compose to select it from the addressbook. 

I don't think I'll get used to this  so a big +1   if this usablity enhancement can be implemented.

I don't like feature creep and I would not want modoboa webclient to become to fat. But this is a small enhancement and a handy one. Not intrusive

**Comment by tonioo** [comment]
@WASDIT @blokkie Do you think this behaviour should be automatic or not?

**Comment by blokkie** [comment]
Yes automatic would be good too but only when  you reply to a mail.
+1

**Comment by WASDIT** [comment]
Yes same meaning like blokkie
